scripts/extractEdition.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from model import Edition
+from main import loadEnvFile
+from managers import DBManager
```

Comment:
For some reason, when I run this script in the scripts folder, I get a ModuleNotFound Error for the model, main, and managers. It works fine when it's outside the scripts folder. Do you have a suggestion as to how to fix this and why it's happening?

Reply:
I think if you leave it in the scripts file and run it from root (like `python main.py scripts/extractEdition.py`) that should work.